src/json/members/krema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Ali Krema",
+  "pennkey": "krema",
+  "major": "Computer Science",
+  "school": "Engineering",
+  "bio": "Catch me either pretending to code or excessively playing FIFA Ultimate Team... or making banana bread.",
+  "hometown": "Cairo, Egypt",
+  "team": "Penn Mobile",
+  "roles": ["Android Mobile Engineer"],
+  "photo": "https://imgur.com/a/zqdcvXD",
```

Comment:
Just one small thing: this is a link to a imgur webpage, not an actual image file. I went to the link and clicked "copy image link" to get this link: https://i.imgur.com/lT1Xivj.jpeg. You can see when you open this link it's an actual image file not a webpage. Could you replace the webpage link you have here with the image link?

Thanks so much! Let me know if you have any questions about this!